editor/block-toolbar/index.js

```diff
@@ -128,6 +99,10 @@ class BlockToolbar extends Component {
 					role="toolbar"
 					deep
 				>
+					<KeyboardShortcuts shortcuts={ {
+						mod: [ this.focusToolbar, true ],
+						'alt+f10': [ this.focusToolbar, true ],
+					} } />
```

Comment:
Nice refactoring 👍

Reply:
Can we handle the "escape" key as well `onToolbarKeyDown`?

Reply:
>Can we handle the "escape" key as well `onToolbarKeyDown`?

We can, but to recreate the handling of these keys only taking effect while within the toolbar, we'd want to introduce child scoping for KeyboardShortcuts. I mentioned this in #1944, and should probably be done as a separate refactoring step.